deploy/olm-catalog/ibm-commonui-operator/1.4.0/ibm-commonui-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -220,6 +484,27 @@ spec:
         - description: Displays names of pods associated with the Legacy Header service
           displayName: Pod Names
           path: nodes
+    - description: NavConfiguration is the Schema for the navconfigurations API
+      kind: NavConfiguration
+      name: navconfigurations.foundation.ibm.com
+      version: v1
+      resources:
+        - kind: NavConfiguration
+          name: ''
+          version: v1
+      specDescriptors:
+        - description: Customizations for the about modal
+          displayName: Navconfig about modal Configuration parameters
+          path: about
+        - description: Customizations for the common web ui header
+          displayName: Navconfig header Configuration parameters
+          path: header
+        - description: Customizations for the login page
+          displayName: Navconfig login Configuration parameters
+          path: login
+        - description: Navigation items for the common web ui header as defined by the cloud paks
```

Comment:
We may need to add statusDescriptors. We need to run this through cv lint and see if it passes. If it doesn't, we need to add a status field and then add the statusDescriptors